Stratis.Bitcoin/Features/MemoryPool/MempoolBehavior.cs

```diff
@@ -107,9 +233,15 @@ protected override void DetachCore()
                 Debugger.Break();
                 throw;
             }
-            
         }
 
+        /// <summary>
+        /// Asynchronous processing of node message.
+        /// Handles the following message payloads: TxPayload, MempoolPayload, GetDataPayload, InvPayload.
+        /// </summary>
+        /// <param name="node">Node sending the message.</param>
+        /// <param name="message">Incoming message.</param>
+        /// <returns></returns>
```

Comment:
You can omit `<returns>` if the function returns `void` or `Task` (but not `Task<T>`)